docs/build/workspace.md

```diff
@@ -0,0 +1,88 @@
+# Tutorial: Integrating multiple packages in a workspace
+
+In this tutorial, we will show you how to integrate multiple pixi packages into a single workspace.
```

Comment:
I think it would be great to have a small description on what it is `workspace` and how it is different from `project`